src/client.h

```diff
@@ -579,6 +579,14 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 		m_client_event_queue.push(event);
 	}
 
+	// Helper function
+	inline std::string getPlayerName()
```

Comment:
const std::string & return to prevent a copy, because we are refering to a player attribute